gems/ruby-saml/OSVDB-124383.yml

```diff
@@ -0,0 +1,11 @@
+---
+gem: ruby-saml
+osvdb: 124383
+url: https://github.com/onelogin/ruby-saml/pull/247
+title: Ruby-Saml Gem is vulnerable to entity expansion attacks
+date: 2015-07-09
```

Comment:
Let's make this 2015-06-30, as that's when https://github.com/onelogin/ruby-saml/commit/a2e5318530701bf14528c5b3b51c880b3499a75d was committed.